app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  belongs_to :family, optional: true
```

Comment:
this ```optional: true``` means that it can belong to something but it can even not belong to that something?

Reply:
Good job